### PR TITLE
ECCB/Bioinformatics 2016 camera-ready experiments

### PR DESCRIPTION
**Pull request by mdml**
I added back the `experiments/eccb2016` scripts and `README.md`, this time for reproducing the experiments in the camera-ready version of the WExT paper.

Run `commands.sh` to reproduce the results from the paper. I tested it quickly by using only 100 permutations. I tested it both with and without the grid.

### CODE REVIEW

File: experiments/eccb2016/scripts/pval_correlations.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            else:
+                t2_pvals = list(df.loc[df['Method'] == t2]['P-value'])
+                rho, pval = spearmanr(t1_pvals, t2_pvals)
+                row.append(rho)
+        tbl.append(row)
+
+    print '-' * 80
+    print 'CORRELATIONS ({})'.format(val)
+    print aligned_plaintext_table('\n'.join([ '\t'.join(map(str, row)) for row in tbl ]) )
+sys.exit()
```
Comment (matthewreyna) [inline experiments/eccb2016/scripts/pval_correlations.py]: Remove `sys.exit()`?

File: weighted_exclusivity_test/constants.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 PTOL = 10**-3
 
 # Set sizes implemented
-WEIGHTED_SADDLEPOINT_SET_SIZES_IMPLEMENTED = set([2, 3])
+WRE_SADDLEPOINT_SET_SIZES_IMPLEMENTED = set([2, 3])
```
Comment (matthewreyna) [inline weighted_exclusivity_test/constants.py]: This should be `WRE_EXACT_SET_SIZES_IMPLEMENTED`.

File: weighted_exclusivity_test/exclusivity_tests.py [hunk 3]
```diff
@@ -25,7 +25,7 @@ def weighted_test(t, x, p, method=EXACT, verbose=0):
     assert(t<=sum(x))
     #Check that we've implemented the given set size with the exact test
     if method == EXACT:
-        assert( len(x) in WEIGHTED_SADDLEPOINT_SET_SIZES_IMPLEMENTED )
+        assert( len(x) in WRE_SADDLEPOINT_SET_SIZES_IMPLEMENTED )
```
Comment (matthewreyna) [inline weighted_exclusivity_test/exclusivity_tests.py]: This should be `WRE_EXACT_SET_SIZES_IMPLEMENTED`.

**Comment by matthewreyna** [comment]
I tested `commands.sh` without the grid (`GRID_ENGINE=false`).  Also using fewer permutations, I produced results that were similar to those in the paper.

I think `commands.sh` would be even better with clearer variable names and more comments.  For example, the differences between the `PAIR_PERMUTATIONS`, `WEIGHT_PERMUTATIONS`, and `EXTRA_PERMUTATIONS` variables might not be as clear to us in the future, and we might wonder why we do not have a `TRIPLE_PERMUTATIONS` variable.

Otherwise, everything looks good to me.

File: weighted_exclusivity_test/exclusivity_tests.py (hunk 3 above)
Comment (mdml) [inline weighted_exclusivity_test/exclusivity_tests.py]: Fixed in f78fcdb.

File: weighted_exclusivity_test/constants.py (hunk 2 above)
Comment (mdml) [inline weighted_exclusivity_test/constants.py]: Fixed in f78fcdb.

File: experiments/eccb2016/scripts/pval_correlations.py (hunk 1 above)
Comment (mdml) [inline experiments/eccb2016/scripts/pval_correlations.py]: Fixed in f78fcdb.

**Comment by mdml** [comment]
@matthewreyna: I made updates as per your suggestions, and renamed as per our discussion on Slack. I then reran everything to make sure it works. Please review and let's finalize and merge!

**Comment by matthewreyna** [comment]
The `commands.sh` script is much clearer now. Everything looks good to me!

**Pull request merged**